kufar/books/main.go

```diff
@@ -166,96 +182,99 @@ func (d Database) SignUpUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write([]byte(http.StatusText(http.StatusOK)))
+	w.Write([]byte(http.StatusText(http.StatusOK) + ": auth passed"))
 }
 
-//SignInUser ...
-func (d Database) SignInUser(w http.ResponseWriter, r *http.Request) {
-	credReq := CredentialsRequest{}
-	if err := json.NewDecoder(r.Body).Decode(&credReq); err != nil {
-		http.Error(w, http.StatusText(http.StatusInternalServerError)+": error unmarshalling json", http.StatusInternalServerError)
-		return
-	}
+func (d Database) getBookUserID(idInt int) (int, error) {
```

Comment:
`idInt` isn't quite good naming. Better to use something like `bookID` (if it's a book id :D)

Reply:
true